phpunit/class-gutenberg-rest-template-controller-test.php

```diff
@@ -160,6 +160,44 @@ public function test_get_item() {
 		);
 	}
 
+	/**
+	 * Ticket 54507
+	 */
+	public function test_get_item_works_with_a_single_slash() {
+		wp_set_current_user( self::$admin_id );
+		$request = new WP_REST_Request( 'GET', '/wp/v2/templates/tt1-blocks/index' );
+		$old_uri = $_SERVER['REQUEST_URI'];
+
+		$_SERVER['REQUEST_URI'] = '/index.php/wp-json/wp/v2/templates/tt1-blocks//index/?context=edit&_locale=user';
```

Comment:
Let's not assume that the site is at the root of the server, i.e. `/foo/bar/index.php/...`